vwifi.c

```diff
@@ -44,10 +48,20 @@ struct owl_ndev_priv_context {
     struct wireless_dev wdev;
 };
 
-static char *ssid_list = "";
+/*
+ * AP information table entry. Entry field is extendable.
```

Comment:
I did not get the meaning of "Entry field is extendable." What did you tend to express?

Reply:
Store other AP information such as Capability information which is from the beacon frame.
I could remove the comment "Entry field is extendable."